pyanaconda/threading.py

```diff
@@ -288,15 +288,15 @@ def run(self):
 
         log.info("Running Thread: %s (%s)", self.name, self.ident)
         try:
-            threading.Thread.run(self)
+            super().run()
         # pylint: disable=bare-except
         except:
             if self._fatal:
                 sys.excepthook(*sys.exc_info())
             else:
                 threadMgr.set_error(self.name, *sys.exc_info())
         finally:
-            threadMgr.remove(self.name)
```

Comment:
You can't do this change here. The `super()` call is not `threadMgr` object in this case.